### DISCUSSION TITLE
Is hubs loading time significantly longer?

### DISCUSSION THREAD

**Discussion by deanmax** [mozilla/hubs#2885] [General Questions & Discussion] [answered]
**Description**
Starting Aug 17, 2020, opening hubs scene takes much longer time to load at the "hubs" logo view(see below)

This happens on both my Mac and iphone. I also tried using incognito mode and saw no difference.

**To Reproduce**
1. open any hubs room(https://hubs.mozilla.com/BVXL5QX/ in my case)
2. you will experience much longer wait time in this view (I'm a patient guy, but still can't bear it)

**Expected behavior**
Load time should be no more than 30 seconds

**Screenshots**
<img width="600" alt="Screen Shot 2020-08-19 at 3 42 00 PM" src="https://user-images.githubusercontent.com/3320547/90682306-c2a42180-e232-11ea-8059-8e07f59c3261.png">

**Hardware** 
 - Device: Macbook Pro 15'' with Radeon Pro 555X 4 GB graphic card
 - OS: Mojavi, iOS 13.1
 - Browser Chrome, Safari

**Additional context**

**Comment by deanmax** [comment]
Update: Firefox seems to work just fine

**Comment by misslivirose** [comment, marked as answer]
I haven't observed this or heard other reports of loading times in Hubs - are there specific rooms or scenes where this happens? Do you have any errors in the developer console (which can be opened with F12)?

**Reply by robinkwilson** [reply]
[Temporarily marked as answered] Unmark this as "answered", when you add more information.

**Comment by deanmax** [comment]
It seems like I'm using a customized scene which has too many contents to load. Switching it back to hubs official scene solved the problem. 👍
